src/redux/books/books.js

```diff
@@ -1,34 +1,81 @@
+// import { createSlice } from '@reduxjs/toolkit';
+import uuid from 'react-uuid';
+
+// action types
 const ADD = 'BOOK_ADDED';
 const REMOVE = 'BOOK_REMOVED';
 
+// Action creators
+export const addBook = ({ title, author }) => ({
+  type: ADD,
+  id: uuid(),
+  title,
+  author,
+});
+
+export const removeBook = (id) => ({
+  type: REMOVE,
+  id,
+});
+
 const getFromAction = ({ id, title, author }) => ({
   id, title, author,
 });
 
-const booksReducer = (state = [], action = []) => {
+/*
+const booksSlice = createSlice({
+  name: 'books',
+  initialState: [
+    {
+      id: 1,
+      title: 'book 1',
+      author: 'author 1',
+    },
+    {
+      id: 2,
+      title: 'book 2',
+      author: 'author 2',
+    }
+  ],
+  reducers: {
+    addBookReducer(state, action) {
+      state.push({
+        id: action.payload.id,
+        title: action.payload.title,
+        author: action.payload.author,
+      })
+    },
+    removeBookReducer(state, action) {
+      state.filter((book) => book.id !== action.payload.id);
+    }
+  }
+});
+*/
```

Comment:
[OPTIONAL] Kindly remove commented code you do not need with respect to the YAGNI principle. This will make your code a lot cleaner and readable.